### PR TITLE
Merge Ulotka informacyjna

### PR DESCRIPTION
**Pull request by lukp**
Please merge this

### CODE REVIEW

File: Ulotka-informacyjna.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Podział kosztów przedstawiony jest w poniższej tabeli
+
+|     KOSZTY/KTO      |   Kursanci   |                  Młodsza Kadra                   | KAdra |
+| :-----------------: | :----------: | :----------------------------------------------: | :---: |
+|       Dojazd        | Swój + kadra | Swój (częściowy zwrot od kursantów wg algorytmu) |
+|       Nocleg        | Swój + kadra |                       Swój                       |
+|   Pływanie Kolna    | Swój + kadra |                       Swój                       |
+| Pływanie Wietrznice | Swój + kadra |                Opłacają kursanci                 |
+
+[aw]: (https://docs.google.com/spreadsheets/d/10giT2T6qyZJoPy82lyKUyZWgP5miSnaLXoyV4FlH55o/edit#gid=71587447)
```
Comment (lukp) [inline Ulotka-informacyjna.md:53]: Very nice way to describe link

File: Ulotka-informacyjna.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+- Doskonalenie techniki wchodzenia i wychodzenia z cofek
+  \*Zachowanie w odwoju
+- Umiejętności czytania wody
+- Wykorzystanie potencjału 66,6% polskich sztucznych torów slalomowych do rozwoju umiejętności kajakarskich
+- Integracja uczestników
+
+## 🗺️ LOKALIZACJA I TERMINY
+
+➡️ **Warszawa**
+
```
Comment (lukp) [inline Ulotka-informacyjna.md:30]: Maybe this would look better in a table?

File: Ulotka-informacyjna.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# HABATORY ROK
```
Comment (lukp) [inline Ulotka-informacyjna.md:1]: Use placeholder for ROK

File: Ulotka-informacyjna.md (hunk 2 above)
Comment (amzdebska) [reply Ulotka-informacyjna.md:30]: ok no

Comment (lukp) [reply Ulotka-informacyjna.md:30]: Why?

**Pull request merged**